### ISSUE TITLE
Support for installation on virtualenv

### ISSUE THREAD

**Issue by doronbehar** [veltzer/templar#1]
Hello @veltzer,

I'm interested in contributing to https://github.com/vertzel/openbook, in order to compile my version of the book, I need to use `templar` and I would like to install it in a `virtualenv`, but I get this error:
```
...
    creating /usr/share/templar
    error: could not create '/usr/share/templar': Permission denied
```

Is there a variable or an option I can change in order to install in my `virtualenv` directory?

**Comment by veltzer** [comment]
You are absolutely right. I need to add support for installing templar in a virtual env. I will work on it....
